src/authentication/authentication.service.ts

```diff
@@ -20,14 +21,61 @@ export class AuthenticationService {
     private logger: Logger,
   ) {}
 
+  bypassUser(userId: string, password: string) {
+    if (
+      this.configService.get<string>('app.env') === 'development' &&
```

Comment:
change logic to not equal to 'production' so that this can be enabled for all non-prod environments if needed